UPGRADE-3.2.md

```diff
@@ -134,7 +134,10 @@ TwigBridge
 ----------
 
  * Deprecated the possibility to inject the Form Twig Renderer into the form
-   extension. Inject it into the `TwigRendererEngine` instead.
+   extension. Inject the Twig Environment into the `TwigRendererEngine` and load 
+   the Twig Renderer using the Twig FactoryRuntimeLoader instead.
```

Comment:
I would use the class name here. And we should probably also mention that you need Twig 1.30 for it.